src/ui/flutter_app/lib/shared/custom_drawer/src/item.dart

```diff
@@ -67,7 +66,7 @@ class CustomDrawerItem<T> extends StatelessWidget {
 
     return InkWell(
       splashColor: AppTheme.drawerSplashColor,
-      highlightColor: AppTheme.drawerHighlightColor,
+      highlightColor: Colors.transparent,
```

Comment:
So we need to remove a configuration `drawerHighlightColor` from the settings accordingly.

Reply:
there isn't any `drawerHighlightColor` in the code. The overley color can still be customized though.